LocalReferenceLinkRel/explainer.md

```diff
@@ -116,16 +118,34 @@ files. Loop detection may be necessary for references to another
     }
   </style>
   <p>Inside Shadow DOM</p>
+    <template shadowrootmode="open">
+      <link rel="stylesheet" href="#inline_styles_from_shadow" />
+      <p>Inside Nested Shadow DOM</p>
+    </template>
 </template>
-<link rel="stylesheet" href="foo.css" id="#inline_styles_from_shadow" />
 <p>
-  Outside Shadow DOM. Styles defined inside the Shadow Root are not applied, so
-  this text is not blue.
+  Styles defined inside the parent Shadow Root are not applied, so "Inside Nested Shadow DOM" is not blue.
 </p>
 ```
```

Comment:
I wonder if it is worth noting that we could look into a way to export/import ids if we find sufficient use cases for these scenarios?

Reply:
Maybe that could go into open questions

Reply:
Good idea, added to Open Questions